app/spiffworkflow/PropertiesPanel/parts/DataObjectSelect.js

```diff
@@ -0,0 +1,76 @@
+import {useService } from 'bpmn-js-properties-panel';
+import { SelectEntry } from '@bpmn-io/properties-panel';
+
+/**
+ * Finds the value of the given type within the extensionElements
+ * given a type of "spiff:preScript", would find it in this, and retnr
+ * the object.
+ *
+ *  <bpmn:
+ <bpmn:userTask id="123" name="My User Task!">
+ <bpmn:extensionElements>
+ <spiff:preScript>
```

Comment:
did you decide that `spiff:` was better than `spiffworkflow:`. if not, i'd vote the latter for branding.

Reply:
Comment was bad, but the code did the right thing.  Fixed.